### DISCUSSION TITLE
Book4_Ch13_Page14

### DISCUSSION THREAD

**Discussion by Hurricane-k** [Visualize-ML/Book4_Power-of-Matrix#10] [General]
Eq.(35)下几何视角：在越读过程中存在一些疑惑，感觉符号上存在一定歧义：

1. 实心散点（不论蓝色还是红色）本质应该是代表矩阵**X**的行向量吧？若干个实心散点代表一个数据矩阵X？本人看到这里存在一定困惑，是一个实心散点代表一个数据矩阵**X**还是若干个实心散点代表一个数据矩阵**X**？
2. 一个实心散点投影到span(vj)的每一个×表示数据矩阵**X**每一个行向量在span(vj)到原点的距离，可以这样理解吗？顺着这个逻辑应该是若干个×构成了一个yj（yj是一个向量？）？
3. 总结：这里的符号感觉有一定歧义，实心散点（蓝色或者红色）同时代表了矩阵本体和矩阵中的某一行向量，×同时代表坐标以及距离（个人直观感觉是距离是标量，坐标是数组，感觉用同一个表达不合适？）

**Comment by Hurricane-k** [comment]
sry 我有仔细越读了一下 您在图8中标明了每一个×表示数据矩阵X中的每一个行向量到span(vj)的投影，所以我个人对×这个符号存在歧义这一点没有了，主要是个人感觉实心散点的表示存在歧义，谢谢

**Comment by Visualize-ML** [comment]
谢谢你的来信。关于数据的行列向量这一点确实很容易混淆。换一个角度，其实从统计角度更容易理解。

鸢尾花数据的前两列，花萼长，花萼宽，实际上就是随机变量X1、X2。数据就是 (X1, X2) 的样本。

平面散点就是 (X1, X2)  样本的可视化方案。而X1、X2本身可以看做是“向量”。

不知道，这种解释是否有助于理解？

欢迎继续批评。Cheers,
James

**Reply by Hurricane-k** [reply]
OK 谢谢您的回答 我自己想了一下 数据矩阵X是根据数据量的多少而变化 可以是“向量”（一个鸢尾花数据）也可以是“矩阵”（若干个鸢尾花数据拼接在一起）想问一下您我这样的理解是否正确

**Comment by Visualize-ML** [comment]
可以这么理解。我在《矩阵力量》提到过，鸢尾花数据集的行向量就是一朵花，列向量就是一个特征。行向量、列向量实际上是矩阵的两种观察视角，可以叫做duality。

Cheers,
James

**Reply by Hurricane-k** [reply]
Thanks so much for your answer. That gives me a much deeper understanding.

Regards,
Carry

**Comment by Visualize-ML** [comment]
感谢你对鸢尾花书的支持，清华社安排了赠书活动。

活动页面：
https://zhuanlan.zhihu.com/p/622091048
